main.nf

```diff
@@ -3418,7 +3418,7 @@ process ControlFreecViz {
     script:
     """
     echo "Shaping CNV files to make sure we can assess significance"
-    LINEWIDTH=`head -1 ${cnvTumor}| wc -w`; awk 'NF=='$LINEWIDTH'{print}' ${cnvTumor} > TUMOR.CNVs
+    LINEWIDTH=`head -1 ${cnvTumor}| wc -w`; awk 'NF=='\$LINEWIDTH'{print}' ${cnvTumor} > TUMOR.CNVs
```

Comment:
I think it is easier to use this sentence:

awk '{print $NF}' ${cnvTumor} > TUMOR.CNVs

Reply:
Actually, that line to reshape the CNVs is not needed. 

https://github.com/BoevaLab/FREEC/blob/master/scripts/assess_significance.R

Is already taking into consideration the shape of the input CNVs.